src/kibana/components/vislib/visualizations/tile_map.js

```diff
@@ -662,8 +900,7 @@ define(function (require) {
     TileMap.prototype.destroy = function () {
       if (this.popups) {
         this.popups.forEach(function (popup) {
-          popup.elem.off('mouseover').off('mouseout');
-          popup.layer.unbindPopup(popup.elem);
+          popup.off('mouseover').off('mouseout');
```

Comment:
`popups` is an array of objects returned from `layer.on`. I'm not 100% on what's happening here, but if this is a leaflet layer (as I'm assuming it is), does `.off()` work this way? 

According [to the docs](http://leafletjs.com/reference.html#events), leaflet's `.off` needs the handler to properly unbind the event.